src/main/resources/db/migration/V163__RemoveBlankStrings.sql

```diff
@@ -0,0 +1,148 @@
+ALTER TABLE public.gbif_taxa ADD CHECK ("nomenclatural_status" NOT SIMILAR TO ' *');
+ALTER TABLE public.gbif_taxa ADD CHECK ("order" NOT SIMILAR TO ' *');
+ALTER TABLE public.gbif_taxa ADD CHECK ("phylum" NOT SIMILAR TO ' *');
+ALTER TABLE public.gbif_taxa ADD CHECK ("specific_epithet" NOT SIMILAR TO ' *');
+ALTER TABLE public.gbif_vernacular_names ADD CHECK ("country_code" NOT SIMILAR TO ' *');
+ALTER TABLE public.gbif_vernacular_names ADD CHECK ("language" NOT SIMILAR TO ' *');
+ALTER TABLE public.organizations ADD CHECK ("country_code" NOT SIMILAR TO ' *');
+ALTER TABLE public.organizations ADD CHECK ("country_subdivision_code" NOT SIMILAR TO ' *');
+ALTER TABLE public.organizations ADD CHECK ("description" NOT SIMILAR TO ' *');
+ALTER TABLE public.species ADD CHECK ("common_name" NOT SIMILAR TO ' *');
+ALTER TABLE public.species ADD CHECK ("family_name" NOT SIMILAR TO ' *');
+ALTER TABLE public.species_problems ADD CHECK ("suggested_value" NOT SIMILAR TO ' *');
+ALTER TABLE public.timeseries ADD CHECK ("units" NOT SIMILAR TO ' *');
+ALTER TABLE public.upload_problems ADD CHECK ("field" NOT SIMILAR TO ' *');
+ALTER TABLE public.upload_problems ADD CHECK ("message" NOT SIMILAR TO ' *');
+ALTER TABLE public.upload_problems ADD CHECK ("value" NOT SIMILAR TO ' *');
+ALTER TABLE public.users ADD CHECK ("auth_id" NOT SIMILAR TO ' *');
+ALTER TABLE public.users ADD CHECK ("first_name" NOT SIMILAR TO ' *');
+ALTER TABLE public.users ADD CHECK ("last_name" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_city" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_country_code" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_country_subdivision" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_landowner" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_name" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("collection_site_notes" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("founder_id" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("number" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.accessions ADD CHECK ("processing_notes" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.bags ADD CHECK ("bag_number" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.viability_tests ADD CHECK ("notes" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.viability_tests ADD CHECK ("staff_responsible" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.withdrawals ADD CHECK ("destination" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.withdrawals ADD CHECK ("notes" NOT SIMILAR TO ' *');
+ALTER TABLE seedbank.withdrawals ADD CHECK ("staff_responsible" NOT SIMILAR TO ' *');
+ALTER TABLE tracking.planting_sites ADD CHECK ("description" NOT SIMILAR TO ' *');
+ALTER TABLE tracking.plantings ADD CHECK ("notes" NOT SIMILAR TO ' *');
```

Comment:
Compiling this list must have been rather painstaking, thanks for doing this.

Reply:
Not too painstaking, happily -- see the comment at the top of the migration file to see how I did it.